data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -623,6 +663,20 @@ private UInt64 findSlotAtEndOfNextEpoch(final UInt64 epoch) {
     return spec.computeStartSlotAtEpoch(epoch.plus(2)).minus(1);
   }
 
+  private Optional<UInt64> findLatestAvailableEpochForRewardCalculation() {
+    final int minNumOfSlots = 2;
+    return recentChainData
+        .getCurrentEpoch()
+        .flatMap(
+            epoch -> {
+              if (epoch.isGreaterThanOrEqualTo(minNumOfSlots)) {
+                return Optional.of(epoch.minus(minNumOfSlots));
```

Comment:
oh so we have to be 2 epochs behind the head epoch?

also `minNumberOfSlots` is not what this is if we're decrementing epoch by it...

Reply:
Yeah, I messed up the name...My brain isn't keeping up with rewards API anymore...